tests/app/data/docker/manifests-delta/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2.3'
```

Comment:
`docker-compose-dev.yml` files are unused. Remove?

Reply:
removed! (it was present in the artifact as a place holder and to show that you can have more than one file, unused here, thank you)